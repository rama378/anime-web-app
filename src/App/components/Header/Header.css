.Header {
  padding: 16px;
  display: flex;
  align-items: center;
  background-color: var(--background-dark);
}

.Header__logo {
  flex-grow: 1;
}
