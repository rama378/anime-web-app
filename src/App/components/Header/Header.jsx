import './Header.css';
import Logo from '../../common/components/Logo/Logo';
import Menu from '../../components/Menu/Menu';
import UserProfile from '../../components/UserProfile/UserProfile';

const Header = () => {
	return (
		<div className='Header'>
			<div className='Header__logo'>
				<Logo />
			</div>
			
			<Menu />
			{/* <UserProfile /> */}
		</div>
	);
};

export default Header;
