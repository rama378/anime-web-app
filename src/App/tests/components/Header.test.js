import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Header from '../../components/Header/Header';

const renderer = new ShallowRenderer();

describe('<Header />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<Header />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
