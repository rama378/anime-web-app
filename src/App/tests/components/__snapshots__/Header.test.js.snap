// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render and match the snapshot 1`] = `
<div
  className="Header"
>
  <div
    className="Header__logo"
  >
    <Logo />
  </div>
  <Menu />
</div>
`;
